home-data-migration/pom.xml

```diff
@@ -105,6 +105,7 @@
                 </executions>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
What is the purpose of this addition? How did it work before?

Reply:
I got an error because the idea didn't pull up the dependencies. But after Invalidate Caches, the error disappeared.
Also in the plugin above <groupId> org.apache.maven.plugins </groupId> is specified, but not here.
Do I need to remove this change?

Reply:
I assume that `<groupId>org.apache.maven.plugins</groupId>` is the default one for maven plugins and explicitly specifying it is redundant isn`t it?